xray/src/bin/merge_xray_quadtrees.rs

```diff
@@ -0,0 +1,23 @@
+use std::path::PathBuf;
+use structopt::StructOpt;
+
+#[derive(StructOpt, Debug)]
+#[structopt(name = "merge_xray_quadtrees")]
+/// Merge partial xray quadtrees. We assume that the root
+/// of each quadtree belongs to the same level of the final
+/// quadtree.
+struct CommandlineArguments {
+    /// Directory where to write the merged quadtree. Does *not*
+    /// have to be disjoint from input_directories.
+    #[structopt(parse(from_os_str), long = "output_directory")]
+    output_directory: PathBuf,
+    /// Directories with, possibly multiple, partial xray quadtrees.
+    #[structopt(parse(from_os_str))]
```

Comment:
We may explicitly add a `long = "input-directory"` here, as this argument can be specified multiple times and then IMHO it reads nicer to have
```
merge_xray_quadtree --input-directory bla --input-directory blub ...
```

Reply:
Do you find `merge_xray_quadtree --output-directory OUT --input-directory IN1 --input-directory IN2` nicer to read than `merge_xray_quadtree --output-directory OUT IN1 IN2`>

Reply:
Actually your second version looks nice, too, so we can use this as well

Reply:
:+1:
I'll go with the second version then :)